src/tape/ensemble.py

```diff
@@ -1107,6 +1108,9 @@ def batch(
             source or object tables. If not specified, then the id column is
             used by default. For TAPE and `light-curve` functions this is
             populated automatically.
+        single_lc: `int`, optional
```

Comment:
I think it would be really helpful to also allow `single_lc=True`, where select_random_lightcurve is used to pick one for the user.

Reply:
Yeah that makes sense. To do that, I added an `id_only` parameter to `Ensemble.select_random_timeseries` so that it will only fetch the object id of a random lightcurve. But let me know if you would prefer that not added to the API and I can break that out differently.

Reply:
Thanks! That looks like a good solution